script/tests/test_array_literals.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
@robotdana you need to add this to https://github.com/penelopezone/rubyfmt/blob/trunk/script/test.sh for it to run in CI. it looks like it's missing right now?